test.js

```diff
@@ -638,3 +639,50 @@ test('basic - unslab checks', function (t) {
     t.ok(pull.state.buffer.byteLength < 100, 'pull.state.buffer no slab')
   })
 })
+
+function udxPair () {
+  const u = new UDX()
+  const socket1 = u.createSocket()
+  const socket2 = u.createSocket()
+  for (const s of [socket1, socket2]) s.bind()
+
+  const stream1 = u.createStream(1)
+  const stream2 = u.createStream(2)
+  stream1.connect(socket1, stream2.id, socket2.address().port, '127.0.0.1')
+  stream2.connect(socket2, stream1.id, socket1.address().port, '127.0.0.1')
+
+  return [
+    new NoiseStream(true, stream1),
+    new NoiseStream(false, stream2),
+
+    async () => {
+      for (const stream of [stream1, stream2]) stream.end()
+      await socket1.close()
+      await socket2.close()
+    }
+  ]
+}
+
+test('encrypted unordered message', async function (t) {
+  const [a, b, destroy] = udxPair()
+  const message = Buffer.from('plaintext', 'utf8')
```

Comment:
b4a

Reply:
test.js does not seem to be using b4a / not imported.

Reply:
i see, lets add a mental todo for that, no worries